cinema/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+    duration = models.IntegerField()
```

Comment:
Consider adding a `help_text` or `verbose_name` to the `duration` field to clarify what unit of time is expected (e.g., minutes, seconds). This can improve the model's readability and usability in the admin interface.